read-the-docs/source/JIRA.rst

```diff
@@ -124,13 +124,31 @@ change_status()
 ---------------
 
 ``change_status(self, status)``
+``change_status(self, status, **kwargs)``
 
 Changes status of a JIRA ticket.
 
+Some JIRA instances might not need extra parameters to change the status of the ticket. These instances will use the below code example.
+
 .. code:: python
 
     t = ticket.change_status('In Progress')
 
+With the future state of JIRA, some JIRA instances may not automatically set resolution when changing the status of the ticket to RESOLVED.
+We need to pass additional fields like 'resolution' or 'comment' to resolve the tickets via automation using TicketUtil itself. See examples below.
+
+Parameters needed for the JIRA ticket to be Resolved from To Do/Groomed
```

Comment:
Remove this line because it's specific to our RH Jira instance.

Reply:
The changes have been incorporated in the latest commit